app/format/format.tsx

```diff
@@ -102,6 +102,12 @@ export function formatTimestampMillis(timestamp: number | Long) {
   return `${moment(+timestamp).format("MMMM Do, YYYY")} at ${moment(+timestamp).format("h:mm:ss a")}`;
 }
 
+export function formatTimestampSecondsNanos(seconds: number | Long, nanos: number | Long) {
+  return `${moment(+seconds * 1000).format("MMMM Do, YYYY")} at ${moment(+seconds * 1000).format(
```

Comment:
I think this could be simplified to 

```
export function formatTimestampSecondsNanos(seconds: number | Long, nanos: number | Long) {
  return `${formatTimestampMillis(seconds * 1000)}:${nanos.toString().substring(0, 2)}`;
}
```
also i'm curious if nanos always just has 2 digits? `.substring(0, 2)` seems a little brittle. can't there be like 1000000000 nanoseconds in a second?

might also be more convenient for the caller to just be able to pass in the timestamp object that has `seconds` and `nanos` as fields (it'd make the calling code much less verbose). i.e. i think you could make the type signature something like `formatTimestamp(timestamp: {seconds: number | Long, nanos: number | Long})` and use timestamp.seconds and timestamp.nanos below. that'd make the calling code much less verbose i.e.
```
format.formatTimestampSecondsNanos(this.state.actionResult.executionMetadata.outputUploadCompletedTimestamp)
```

Reply:
Yeah, I added 2 more digits from the nanos but past that, it usually just 0s

Reply:
also taking in the timestamp object like that in the parameter really simplified the code, ty!

Reply:
the timestamp object changes look great!

i'm not sure the substring changes are doing exactly what you'd expect. for example if one nanos value was 100 and the other was 1000 - both of these would show "10" using this method 

i feel like what we actually want is something like Math.floor(timestamp.nanos / 1000000)